VAGRANT.md

```diff
@@ -0,0 +1,167 @@
+# Install Nominatim in a virtual machine for development and testing
+
+This document describes how you can install Nominatim inside a Ubuntu 14
+virtual machine on your desktop/laptop (host machine). The goal is to give
+you a development environment to easily edit code and run the test suite
+without affecting the rest of your system. 
+
+The installation can run largely unsupervised. You should expect 1-2h from
+start to finish depending on how fast your computer and download speed
+is.
+
+## Prerequisites
+
+1. [Virtualbox](https://www.virtualbox.org/wiki/Downloads)
+
+2. [Vagrant](https://www.vagrantup.com/downloads.html)
+
+3. Nominatim 
+
+        git clone --recursive https://github.com/twain47/Nominatim.git
+
+    If you haven't used `--recursive`, then you can load the submodules using
+    
+        git submodule init
+        git submodule update
+
+
+
+## Installation
+
+1. Start the virtual machine
+
+        vagrant up
+
+2. Log into the virtual machine
+
+        vagrant ssh
+
+3. Import a small country (Monaco)
+
+    You need to give the virtual machine more memory (2GB) for an import, see `Vagrantfile`.
+    
+    See the FAQ how to skip this step and point Nominatim to an existing database.
+
+  ```
+  # inside the virtual machine:
+  cd Nominatim
+    wget --no-verbose --output-document=data/monaco.osm.pbf http://download.geofabrik.de/europe/monaco-latest.osm.pbf
+    utils/setup.php --osm-file data/monaco.osm.pbf --osm2pgsql-cache 1000 --all | tee monaco.$$.log
```

Comment:
I would recommend also catching errors, i.e "... -all 2>&1 | tee monaco.$$.log".

Also, there is a "./" missing at the beginning.